apollo-router/src/logging/mod.rs

```diff
@@ -97,7 +102,21 @@ macro_rules! assert_snapshot_subscriber {
     };
 
     ($redactions:tt) => {
-        $crate::logging::test::SnapshotSubscriber::create_subscriber(|yaml| {
+        $crate::logging::test::SnapshotSubscriber::create_subscriber(tracing_core::metadata::LevelFilter::INFO, |yaml| {
+            insta::with_settings!({sort_maps => true}, {
+                // the tests here will force maps to sort
+                let mut settings = insta::Settings::clone_current();
+                settings.set_snapshot_suffix("logs");
+                settings.set_sort_maps(true);
+                settings.bind(|| {
+                    insta::assert_yaml_snapshot!(yaml, $redactions);
+                });
+            });
+        })
+    };
+
+    ($redactions:tt, $level:tt) => {
+        $crate::logging::test::SnapshotSubscriber::create_subscriber($level, |yaml| {
```

Comment:
I think this can be DRYed up a bit with just one implementation and the other macro calls delegating to it.

Reply:
i think i did this, please let me know if you meant something else!